source/priority_queue.c

```diff
@@ -169,6 +169,8 @@ int aws_priority_queue_init_dynamic(
     int ret = aws_array_list_init_dynamic(&queue->container, alloc, default_size, item_size);
     if (ret == AWS_OP_SUCCESS) {
         AWS_POSTCONDITION(aws_priority_queue_is_valid(queue));
+    } else {
+        AWS_POSTCONDITION(aws_priority_queue_is_wiped(queue));
```

Comment:
Change to use macros from https://github.com/awslabs/aws-c-common/pull/453

Reply:
Now using the AWS_IS_ZEROED macro